docker-compose.yml

```
@@ -30,6 +35,11 @@ services:
     environment:
       - POSTGRES_EXTENSIONS=citext
       - POSTGRES_HOST_AUTH_METHOD=trust
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 1s
+      timeout: 30s
+      retries: 30
```

This spawns a separate shell, causing extra memory usage.

Let's make this more efficient:

```suggestion
    healthcheck:
      test: ["CMD", "pg_isready", "-U", "postgres"]
      interval: 1s
      timeout: 30s
      retries: 30
```